### PR TITLE
[sio-model] Patient status shex and example rdf

### PR DESCRIPTION
**Pull request by rajaram5**
Patient status shex and example rdf files are added. Did some structure cleaning like folder renaming.

### CODE REVIEW

File: shex/sio-model/patientStatusShape.shex [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
+PREFIX hl7: <http://purl.bioontology.org/ontology/HL7/>
+
+
+:personShape IRI {
+  a [obo:NCBITaxon_9606];
+  sio:SIO_000228 @:personRoleShape;
```
Comment (sxzhang1201) [inline shex/sio-model/patientStatusShape.shex:11]: The cardinality here is 1 to 1, which is not consistent with the multi-role in the diagram.

File: shex/sio-model/patientStatusShape.shex [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+   sio:SIO_000356 @:statusProcessShape; 
+   sio:SIO_000356 @:deadDateProcessShape) 
+} 
+
+:statusProcessShape IRI {
+  sio:SIO_000229 @:outputShape
+}
+
+:outputShape IRI {
+  a [obo:PATO_0001422 obo:PATO_0001421  snomedct:399307001 hl7:C4291647]
```
Comment (sxzhang1201) [inline shex/sio-model/patientStatusShape.shex:29]: Question about the diagram: 
why the type `sio:realizable-entity` is not attached to the status node MV1? 
I see both **_gender_** and _**birth date**_ in the _**Personal Information**_ Module are attached.

File: example-data/turtle/sio-model/patientStatusDead.ttl [hunk 3]
```diff
@@ -0,0 +1,35 @@
+@prefix : <http://purl.org/ejp-rd/cde/v020/example-rdf/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix sio: <http://semanticscience.org/resource/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:person1
+  a obo:NCBITaxon_9606;
+  sio:SIO_000228 :role1;
```
Comment (sxzhang1201) [inline example-data/turtle/sio-model/patientStatusDead.ttl:8]: It reads like you choose the use case "A person has a role, and that role participates in two processes", which is nice. But the diagram seems to assign the status and the death date into two separate processes (though I think the two tasks can be done at the same time in the same process in practice~).

File: shex/sio-model/patientStatusShape.shex (hunk 1 above)
Comment (rajaram5) [reply shex/sio-model/patientStatusShape.shex:11]: Yes, I agree but I am not sure if the diagram is correct ;-) lets discuss this with @markwilkinson

File: shex/sio-model/patientStatusShape.shex (hunk 2 above)
Comment (rajaram5) [reply shex/sio-model/patientStatusShape.shex:29]: I think @markwilkinson might left this out in the diagram to improve the readability of the diagram. But I also wonder if we can make `owl file` and add all MV `rdf:type` as children of  `sio:relizable-entity`. What do you think about this solution @markwilkinson ?

Comment (sxzhang1201) [reply shex/sio-model/patientStatusShape.shex:24]: That sounds good to me.

**Pull request merged**